spec/datetimeformat.html

```diff
@@ -168,8 +168,12 @@ <h1>ToDateTimeOptions ( _options_, _required_, _defaults_ )</h1>
       </p>
 
       <emu-alg>
-        1. If _options_ is *undefined*, let _options_ be *null*; otherwise let _options_ be ? ToObject(_options_).
-        1. Let _options_ be ObjectCreate(_options_).
+        1. If _options_ is *undefined*, then
+          1. Let _options_ be ObjectCreate(*null*).
```

Comment:
wouldn't this want to be `fallback`?

Reply:
I guess that would be shorter!

Reply:
ah, i didn’t expand the diff, i just assumed since the options AO had one